cms/templates/settings_graders.html

```diff
@@ -58,7 +58,7 @@ <h2 class="title-2">${_("Overall Grade Range")}</h2>
           <ol class="list-input">
             <li class="field" id="field-course-grading-range">
               <div class="grade-controls course-grading-range well">
-                <a href="#" class="new-grade-button"><span class="icon fa fa-plus"></span></a>
+                <a href="#" class="new-grade-button"><span class="icon fa fa-plus" aria-hidden="true"></span></a>
```

Comment:
sr text? This is the Settings, Grading page in Studio. 
![image](https://cloud.githubusercontent.com/assets/484484/15759275/17052f40-28dc-11e6-9415-2bc11ee5fe1b.png)